frontend/src/metabase/admin/performance/components/StrategyForm.tsx

```diff
@@ -422,19 +422,9 @@ const MultiplierFieldSubtitle = () => (
   <div>
     {t`To determine how long each cached result should stick around, we take that query's average execution time and multiply that by what you input here. The result is how many seconds the cache should remain valid for.`}{" "}
     <Tooltip
-      events={{
-        hover: true,
-        focus: true,
-        touch: true,
-      }}
+      variant="multiline"
       inline={true}
-      styles={{
-        tooltip: {
-          whiteSpace: "normal",
-        },
-      }}
```

Comment:
This PR introduces a `multiline` variant for this Tooltip, which means "normal whitespace, 20rem max-width", so we're now able to use that here